python/Makefile.am

```diff
@@ -1,26 +1,33 @@
 SWIG_SRC = meep.i numpy.i vec.i
 EXTRA_DIST = $(SWIG_SRC)
-HPPFILES=	                          \
- $(top_srcdir)/src/meep.hpp               \
- $(top_srcdir)/src/meep_internals.hpp     \
- $(top_srcdir)/src/bicgstab.hpp           \
- $(top_srcdir)/src/meep/vec.hpp           \
- $(top_srcdir)/src/meep/mympi.hpp
+HPPFILES=                                    \
+ $(top_srcdir)/src/meep_internals.hpp        \
+ $(top_srcdir)/src/bicgstab.hpp              \
+ $(top_srcdir)/src/meep/vec.hpp              \
+ $(top_srcdir)/src/meep/mympi.hpp            \
+ $(top_srcdir)/libmeepgeom/meepgeom.hpp      \
+ $(top_srcdir)/libmeepgeom/material_data.hpp 
 
 BUILT_SOURCES = meep-python.cpp meep.py
 
 CLEANFILES = $(BUILT_SOURCES) meep-python.h
 
 AM_CPPFLAGS = -I$(top_srcdir)/src                   \
+              -I$(top_srcdir)/libmeepgeom           \
               -I$(top_builddir) # for config.h
 
 _meep_la_SOURCES = meep-python.cpp
-_meep_la_LIBADD = $(top_builddir)/src/libmeep@MEEP_SUFFIX@.la $(PYTHON_LIBS)
+_meep_la_LIBADD = $(top_builddir)/src/libmeep@MEEP_SUFFIX@.la -L$(top_srcdir)/libmeepgeom/.libs $(PYTHON_LIBS) -lctlgeom -lmeepgeom
```

Comment:
Should be in `top_builddir`, not `top_srcdir` (in case you are building out-of-tree).

Shouldn't it just be `$(top_builddir)/src/libmeepgeom.la` and omit the `-lmeepgeom`?